appveyor.yml

```diff
@@ -1,47 +1,50 @@
 # Set up the build environment.
 environment:
-    # WORKAROUND START #
-    RUSTUP_USE_HYPER: 1
-    CARGO_HTTP_CHECK_REVOKE: false
-    # WORKAROUND END #
-
-    matrix:
-        - channel: stable
-          target: x86_64-pc-windows-msvc
-        - channel: beta
-          target: x86_64-pc-windows-msvc
-        - channel: nightly
-          target: x86_64-pc-windows-msvc
+  # WORKAROUND START #
+  RUSTUP_USE_HYPER: 1
+  CARGO_HTTP_CHECK_REVOKE: false
+  # WORKAROUND END #
 
+# Configure the build matrix.
 matrix:
-    allow_failures:
-        - channel: nightly
+  - channel: stable
+    target: x86_64-pc-windows-msvc
+  - channel: beta
+    target: x86_64-pc-windows-msvc
+  - channel: nightly
+    target: x86_64-pc-windows-msvc
+
+  # Allow for occasional compiler breakage on nightly Rust.
+  allow_failures:
```

Comment:
stick a `-` here

Error parsing appveyor.yml: (Line: 18, Col: 3, Idx: 410) - (Line: 18, Col: 3, Idx: 410): While parsing a block collection, did not find expected '-' indicator.